### PR TITLE
Adding withdraw delay information

### PR DESCRIPTION
**Pull request by githubdoramon**

### CODE REVIEW

File: docs/dev/troubleshooting/withdraw-delay.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+To introduce this time lock no changes were made in the audited smart contracts. Instead we have used an existing Validator role that we control and additionally restricted ourselves by pointing it at an intermediate smart contract with a time lock. It will be initially configured for a **24 hours** delay, which will be gradually decreased as the system matures. Changing the delay requires multiple signatures collected from several cold wallets owned by zkSync leadership.
+
+This design has the following advantages:
+
+- Even if an attacker finds a critical bug in ZK circuits and also successfully compromises the servers running our sequencer, there is plenty of time to detect an exploit, investigate, and freeze the protocol via governance.
+- No changed introduced into the zkSync Era contracts, so even if the intermediate contract is compromised we revert back to the original state.
```
Comment (anton-matterlabs) [inline docs/dev/troubleshooting/withdraw-delay.md]:
```suggestion
- No changes were introduced into the zkSync Era contracts, so even if the intermediate contract is compromised we revert back to the original state.
```

File: docs/dev/troubleshooting/withdraw-delay.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Withdraw delay
+
+In order to prevent a quick drain of the protocol in case a critical bug has been discovered and exploited, we are introducing a block execution delay. Each L2 block committed to L1 will have a time lock before it is executed and finalized. This means that there is enough time to verify the effects of the transactions included into a block before the block becomes final. The zkSync team will be monitoring each block and be able to investigate any anomaly (e.g. rapid outflow, unusually large withdrawals, etc).
```
Comment (krofax) [inline docs/dev/troubleshooting/withdraw-delay.md:3]: It ought to be "Withdrawal delay" ? @githubdoramon

Comment (githubdoramon) [reply docs/dev/troubleshooting/withdraw-delay.md:3]: indeed.... my bad on this one

File: docs/dev/troubleshooting/withdraw-delay.md (hunk 1 above)
Comment (anton-matterlabs) [inline docs/dev/troubleshooting/withdraw-delay.md]:
```suggestion
- No changes were introduced into the zkSync Era contracts, so even if the intermediate contract is compromised we revert back to the original state.
```

**Review by anton-matterlabs — APPROVED** [review]

**Pull request merged**